zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/cache/process/CachedProcessEntity.java

```diff
@@ -8,5 +8,10 @@
 package io.camunda.exporter.cache.process;
 
 import java.util.List;
+import java.util.Map;
 
-public record CachedProcessEntity(String name, String versionTag, List<String> callElementIds) {}
+public record CachedProcessEntity(
```

Comment:
Just a bidirectional FYI, @EuroLew is also making a change to this record in another PR. One of you will have some conflict resolution related to that